### PR TITLE
adjust outbound response to look for price in res.body

### PR DESCRIPTION
**Pull request by chasingSublimity**

### CODE REVIEW

File: src/outbound.coffee [hunk 1]
```diff
@@ -91,8 +91,8 @@ response = (vars, req, res) ->
     event =
       outcome: vars.default_outcome or 'error'
       reason: event.reason or event.message or 'Unrecognized response'
-  
-  event.price = if (vars.price) then vars.price.valueOf() else 0
+
+  event.price = if (event.price) then parseFloat(event.price) else 0
```
Comment (alexkwolfe) [inline src/outbound.coffee]: I don't think you need this line at all, do you?

Comment (chasingSublimity) [reply src/outbound.coffee]: Price xml data (`<price>1.5</price>`) is parsed as a string in `response`. The `parseFloat` call ensures that we pass a number back.

We should be using `.valueOf()` on all of the `request` functions, so price data like `$1.5` should be normalized by the time it hits `response.

If we are always going to _send_ a default price of `0`, the last condition of that ternary operator probably isn’t necessary, though.

Comment (alexkwolfe) [reply src/outbound.coffee]: The `number` type should convert the string into a number.

Comment (chasingSublimity) [reply src/outbound.coffee]: derp -_-

**Pull request merged**